auth0.md

```diff
@@ -0,0 +1,18 @@
+const { auth } = require('express-openid-connect');
```

Comment:
Delete this file since we're no longer using auth0

Reply:
deleted